appinventor/appengine/src/com/google/appinventor/client/widgets/properties/ChoicePropertyEditor.java

```diff
@@ -101,6 +101,31 @@ public void execute() {
     initWidget(dropDownButton);
   }
 
+  // For theme choice property editor
+  public ChoicePropertyEditor(Choice[] choices, Boolean projectLoaded) {
```

Comment:
Technically it is better to use the primitive `boolean` here rather than the boxed `Boolean` type.

Reply:
I feel like this could be refactored with the other constructor, which effectively is when `projectLoaded = true`.